lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -172,7 +172,7 @@ sub start
    my $config_path = $self->{paths}{config} = $self->write_yaml_file( "config.yaml" => {
         server_name => $self->server_name,
         log_config => $log_config_file,
-        public_baseurl => "http://${bind_host}:$unsecure_port",
+        public_baseurl => "https://${bind_host}:$secure_port",
```

Comment:
I do wonder if this has to take into account the `$WANT_TLS` option somewhere? 🤷

Reply:
Does it work with `--no-tls` still with this? Can we use the unsecure port if its enabled or does that break stuff?

Reply:
TIL that was a flag. 😄 It did break with that, I pushed a commit fixing it.

Reply:
Me too! I was like "WTF is WANT_TLS??"